fall18/p1/main.py

```diff
@@ -0,0 +1,194 @@
+# This line is a comment because it starts with a pound sign (#). That
+# means Python ignores it.  A comment is just for a human reading the
+# code.  This project involve 20 small problems to give you practice
+# with operators, types, and boolean logic.  We'll give you directions
+# (as comments) on what to do for each problem.
+
+############################################################
+print('Problem 1')
+
+# we did this one for you!
+
+print('one plus one is:')
+print(1+1)
+print()
+
+############################################################
+print('Problem 2')
+
+# now fix this one so it prints 4 by adding 2 with 2.  If you just
+# print 4, you may pass the tests and initially get a good grade
+# automatically, but we'll take away those points later when a TA
+# reads your solution!  (this example about grading applies
+# generally).
+
+print('two plus two is:')
+print(0)
+print()
+
+############################################################
+print('Problem 3')
+
+# we did this one for you too!
+
+print('the type of 5 is a(n):')
+print(type(5))
+print()
+
+############################################################
+print('Problem 4')
+
+print('the type of 1.5 is a(n):')
+
+# put something in the parenthesis here to get an answer
+print()
+print()
+
+############################################################
+print('Problem 5')
+
+print('the type of 5.0 is a(n):')
+
+# put something in the parenthesis here to get an answer
+print()
+print()
+
+############################################################
+print('Problem 6')
+
+print('the type of "5.0" is a(n):') # note the quotes!
+
+# put something in the parenthesis here to get an answer
+print()
+print()
+
+############################################################
+print('Problem 7')
+
+print('the type "True" is a(n):')
+
+# put something in the parenthesis here to get an answer
+print()
+print()
+
+############################################################
+print('Problem 8')
+
+print('the type False is a(n):')
+
+# put something in the parenthesis here to get an answer
+print()
+print()
+
+############################################################
+print('Problem 9')
+
+print('the type 2>2 is a(n):')
+
+# put something in the parenthesis here to get an answer
+print()
+print()
+
+############################################################
+print('Problem 10')
+
+print('three plus four is:')
+print("3" + "4") # fix this to get 7
```

Comment:
Would it be appropriate for students to just do print(7) or do we want to make sure they specifically do print(4 + 3)